spec/requests/posts_request_spec.rb

```diff
@@ -2,4 +2,51 @@
 
 RSpec.describe "Posts", type: :request do
 let (:user) {User.create email: 'test@test.com', password:'test111'}
+
+# index
+it 'shows all posts' do
+    Post.create user_id: user.id, pic: 'www.picture.com', des: 'random picture'
+    get '/posts'
+    posts = JSON.parse(response.body)
+
+    expect(posts.length).to eq 1
+    expect(response).to have_http_status(200)
```

Comment:
when you have a chance, it'd be good to add assertions on properties for the post in the response. for example: `expect(posts[0].pic).to eq 'www.picture.com'`